Make bwbcoverbot use cover info from ol dumps (#258)
internetarchive/openlibrary-bots: BWBCoverBot/README.md

# BWBCoverBot
A script to update covers on Open Library using a zip of covers from BWB.

# Setup
Export `OL_ACCESS_KEY` and `OL_SECRET_KEY` (s3) used by the [BWBImportBot](https://openlibrary.org/people/bwbimportbot) account to import.

# Use
Run `python main.py bwbcoverfile.zip`. E.g. `python main.py Apr2021_17_lc_13.zip`.

# How it works

> Each zip file from BWB contains hundreds of `.jpg` files, each prefixed with an ISBN 13 (e.g. `9781483457550.jpg`).

Reviewer: ```suggestion
Each zip file from BWB contains hundreds of `.jpg` files.  The root of each filename is an ISBN 13 (e.g. `9781483457550.jpg`).
```

Author: I made this change.

> BWBCoverBot goes through the zip and for each `.jpg` it checks `./bwb-cover-bot.sqlite` to see if the associated ISBN 13 from the `.jpg`'s filename is in the DB, and if `cover_exists` is 1.
> 
> If the cover is already in the DB, BWBCoverBot moves on, but if it's not, it will query the Open Library API, check if there is a cover, add the cover if necessary, and update `cover_exists`.

Reviewer: ```suggestion
BWBCoverBot goes through the zip and for each `.jpg` it checks `./bwb-cover-bot.sqlite` to see if the associated ISBN 13 from the filename is in the DB, and if `cover_exists` is not 1.

If the cover is not already in the DB, BWBCoverBot will query the Open Library API, check if there is a cover, add the cover if necessary, and set `cover_exists` to 1.
```

Author: I made this change.
